onboarding.md

```diff
@@ -0,0 +1,195 @@
+# Onboarding
+
+>*Note:* Before completing this guide, make sure you have completed the _general_ onboarding guide in the [base mojaloop repository](https://github.com/mojaloop/mojaloop/blob/master/onboarding.md#mojaloop-onboarding).
+
+## Contents
+
+<!-- vscode-markdown-toc -->
+1. [Prerequisites](#Prerequisites)
+2. [Installing and Building](#InstallingandBuilding)
+3. [Running Locally](#RunningLocally)
+4. [Running Inside Docker](#RunningInsideDocker)
+5. [Testing](#Testing)
+6. [Common Errors/FAQs](#CommonErrorsFAQs)
+
+<!-- vscode-markdown-toc-config
+	numbering=true
+	autoSave=true
+	/vscode-markdown-toc-config -->
+<!-- /vscode-markdown-toc -->
+
+#  1. <a name='Prerequisites'></a>Prerequisites
+
+If you have followed the [general onboarding guide](https://github.com/mojaloop/mojaloop/blob/master/onboarding.md#mojaloop-onboarding), you should already have the following cli tools installed:
+
+* `brew` (macOS), [todo: windows package manager]
+* `curl`, `wget`
+* `docker` + `docker-compose`
+* `node`, `npm` and (optionally) `nvm`
+
+In addition to the above cli tools, you will need to install the following to build and run the `email-notifier`:
+
+
+###  1.1. <a name='macOS'></a>macOS
+```bash
+#none - you have everything you need!
+```
+
+###  1.2. <a name='Linux'></a>Linux
+
+[todo]
+
+###  1.3. <a name='Windows'></a>Windows
+
+[todo]
+
+
+##  2. <a name='InstallingandBuilding'></a>Installing and Building
+
+Firstly, clone your fork of the `email-notifier` onto your local machine:
+```bash
+git clone https://github.com/<your_username>/email-notifier.git
+```
+
+Then `cd` into the directory and install the node modules:
+```bash
+cd email-notifier
+npm install
+```
+
+> If you run into problems running `npm install`, make sure to check out the [Common Errors/FAQs](#CommonErrorsFAQs) below.
+
+##  3. <a name='RunningLocally'></a>Running Locally (with dependencies inside of docker)
+
+In this method, we will run core dependencies (`mongodb`, `kafka`) inside of docker containers, while running the `email-notifier` server on your local machine.
+
+> Alternatively, you can run the `email-notifier` inside of `docker-compose` with the rest of the dependencies to make the setup a little easier: [Running Inside Docker](#RunningInsideDocker).
+
+**1. Run the dependencies in `docker-compose`:**
```

Comment:
There is no `docker-compose.yml` for this repo, so it either needs to be written, or perhaps remove these steps.